python/cuml/cuml/dask/preprocessing/label.py

```diff
@@ -106,12 +104,12 @@ def _func_unique_classes(y):
         return rmm_cupy_ary(cp.unique, y)
 
     @staticmethod
-    def _func_xform(model, y):
+    def _func_xform(y, *, model):
```

Comment:
The order is changed here to support dask's `blockswise` / `map_blocks`, which passes partitions of arrays by position and other arguments by keyword.